app/request_schemas/v1/contact_request_schema.rb

```diff
@@ -0,0 +1,39 @@
+module V1
+  class ContactRequestSchema < BaseRequestSchema
+    params do
+      required(:data).value(:hash).schema do
+        required(:type).filled(:str?, eql?: "contact")
+        required(:attributes).value(:hash).schema do
+          required(:msisdn).filled(:string)
```

Comment:
We should rename msidn to phone_number so for the new API we might as well start using phone_number everywhere. We only need a temporary mapping to msisdn when we write to the database and in the serializer

Reply:
Yes, but I want to do this in another migration file renaming this column and also the table name. Rename now will break other parts, so I want to focus on making all structures in place, so when we're renaming, we can have tests to verify it.